### DISCUSSION TITLE
"Space" where taskbar occupies remains after moving taskbar

### DISCUSSION THREAD

**Discussion by PINPAL** [valinet/ExplorerPatcher#1206] [General]
When moving the taskbar to a non-primary display the "space" where the taskbar occupies is initially gone and moves to the new taskbar location however when entering a fullscreen application, sleeping, hibernating or changing resolution this "space" returns to the primary monitor.

# Steps to Reproduce
1. Move taskbar to non-primary display
2. Observe space where the taskbar occupies is now able to be occupied by maximised windows
3. Open fullscreen game.
4. Close the game.
5. Observe when maximising a window the "space" where the taskbar would normally occupy is now showing the desktop despite the taskbar not being on that display.

# Screenshots
### Primary Display
Primary display is showing the "empty space" where the taskbar occupies
![Primary Display](https://user-images.githubusercontent.com/18197217/228128604-a09ba2e2-5ec2-4861-85bd-7d6d939d4af7.png)

### Secondary Display
Secondary display is showing where the taskbar actually is despite the fact it takes up no "space" and maximised windows go behind it
![Secondary Display](https://user-images.githubusercontent.com/18197217/228129084-1a16a820-bbb9-466b-8559-7d0602b318e4.png)

# Workaround
You can temporarily fix the issue by moving the taskbar back to the primary display and then moving it back to your secondary display however this quickly becomes tedious as it occurs whenever opening fullscreen games or putting the system to sleep as previosly mentioned.

**Comment by Geobert** [comment]
I noticed that as well, quite anoying :-/

**Comment by mortismint** [comment]
Yea i hope that gets fixed, it only happens after i start a game in fullscreen.. windows on my second monitor overlay with the taskbar which also sucks.. ):

**Comment by Psycho-Bill** [comment]
newest build and even on primary monitor fullscreen apps open behind taskbar leaving the bottom section where it defaults to as open space after exiting a game.
